### PR TITLE
added the community profile for  Nishant 

### LINKED ISSUE #3054: [Community] Member Profile: Nishant Mittal
**Current Behavior**

@nishantwrp  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @nishantwrp  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @nishantwrp 
- Twitter: nishantwrp
- LinkedIn: nishantwrp
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by legendarykamal**
Signed-off-by: Kamal Nayan <geniusamansingh@gmail.com>

**Description**
image

This PR fixes #3054 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@legendarykamal heads-up the build check is failing. 
```
YAMLException:can not read a block mapping entry; a multiline key may not be   an implicit key at line 13, column 7:
      status: Active
            ^/home/runner/work/layer5/layer5/src/collections/members/nishant/ind  ex.mdx: can not read a block mapping entry; a multiline key may not be an impl  icit key at line 13, column 7:
      status: Active
            ^
```

**Comment by legendarykamal** [comment]
let me see @Chadha93 :eye:

**Comment by l5io** [comment]
🚀 Preview for commit a18341b2533fcee6044f8ce6117c55fd0e9bb58d at: https://62f646062825550cd738d767--layer5.netlify.app

**Review by Chadha93 — APPROVED** [review]
LGTM

File: src/collections/members/nishant/index.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: Nishant
```
Comment (Chadha93) [inline src/collections/members/nishant/index.mdx:2]:
```suggestion
name: Nishant Mittal
```

**Comment by legendarykamal** [comment]
done @Chadha93 , @nishantwrp If this looks good pls approve :100:

**Comment by l5io** [comment]
🚀 Preview for commit 5cdd0a5e353946a28d089b67282946cc72b11a9f at: https://62f6663e64deab22db24ebb8--layer5.netlify.app

**Review by nishantwrp — APPROVED** [review]
Thank you @legendarykamal! 🎉

**Comment by leecalcote** [comment]
Whoo-hoo! @nishantwrp 🎈 🥳

**Comment by legendarykamal** [comment]
ok so sweets distributed :smile: : but who's gonna merge this :sweat_smile: ?

**Pull request merged**